schemas/registry-completions.schema.json

```diff
@@ -11,9 +11,11 @@
   "properties": {
     "version": {
       "description": "The version of the schema document.",
-      "type": "number",
+      "type": "integer",
```

Comment:
I did not know there was an integer type in json schema.

Reply:
I didn't either until I looked up how to constrain the value.